conda_forge_tick/auto_tick.xsh

```diff
@@ -446,11 +446,8 @@ def initialize_migrators(do_rebuild=False):
 
     add_arch_migrate($MIGRATORS,gx)
     add_rebuild_openssl($MIGRATORS, gx)
-    add_rebuild_libprotobuf($MIGRATORS, gx)
-    add_rebuild_blas($MIGRATORS, gx)
```

Comment:
There are code bits associated with these migrations. Should we remove them too or should we leave them there for future re-use?

Reply:
We should keep them around for now